Sources/Extensions/UIKit/UIViewControllerExtensions.swift

```diff
@@ -77,7 +77,68 @@ public extension UIViewController {
         present(alertController, animated: true, completion: nil)
         return alertController
     }
-
+    /// SwifterSwift: Replace the "navigationController?.pushViewController" method.
+    ///
+    /// - Parameters:
+    ///   - viewController: viewController.
+    ///   - animated: (Optional)The default value of “true”.
+    public func push(_ viewController: UIViewController, animated: Bool = true) {
+        navigationController?.pushViewController(viewController, animated: animated)
+    }
+    /// SwifterSwift: Replace the "navigationController?.popViewController" method.
+    ///
+    /// - Parameters:
+    ///   - animated: (Optional)The default value of “true”.
+    @discardableResult public func pop(animated: Bool = true) -> UIViewController? {
+        return navigationController?.popViewController(animated: animated)
```

Comment:
Hey @WangWenzhuang, thanks for taking the time to submit this PR! 👍 Not sure how I feel about these extensions though 🤔 I think it hides the fact that we're in a navigation controller. It also doesn't give us any indication that the method could potentially fail 😢